.github/workflows/doc-build.yml

```diff
@@ -15,14 +15,13 @@ jobs:
   build-docs:
     runs-on: ubuntu-latest
     container: ghcr.io/sagemath/sage/sage-docker-ubuntu-focal-standard-with-targets:dev
-    env:
-      CAN_DEPLOY: ${{ secrets.NETLIFY_AUTH_TOKEN != '' }}
```

Comment:
I think this conditionalization should not be dropped but moved into the new `doc-publish` workflow.
We don't want the `doc-publish` workflow to fail on users' repos that have GH Actions enabled.

Reply:
> We don't want the doc-publish workflow to fail on users' repos that have GH Actions enabled.

Why not? It's not doing its intended job and a red workflow doesn't hurt anyone. Don't you think its better to be notified by the failing workflow in case the auth token is not config correctly in the main repo?

Reply:
If you want to have this kind of monitoring in or main repo, restrict the error to that repo only.

Reply:
Thanks for your suggestion, but I don't plan to invest more time into what feels like an edge case. If you (or someone else) gets annoyed by the red workflow and you don't want to setup your own netlify deploy, you can simply disable this workflow in the github ui (or change the config in your default branch).

Reply:
It was already implemented.
Just don't cut it out.